packages/twenty-server/src/engine/api/graphql/workspace-query-runner/workspace-query-runner.service.ts

```diff
@@ -481,7 +493,12 @@ export class WorkspaceQueryRunnerService {
       throw error;
     }
 
-    return parseResult(result);
+    const resultWithGetters = await this.queryResultGettersFactory.create(
```

Comment:
note: This won't work for nested objects

Reply:
Yep, we may need a recursive approach at some point